gazebo/kinetic-gazebo9/post_rosdep_install.sh

```diff
@@ -29,6 +29,7 @@ echo "Gazebo 7 uninstalled with success"
 
 echo "Installing Gazebo 9"
 sudo apt-get install -y ros-kinetic-gazebo9-*
+rosdep update
```

Comment:
I think you should move this up to before the `sudo apt-get update`.

Reply:
done